applications/API/serializers.py

```
@@ -0,0 +1,505 @@
+        return super().to_representation(data)
+
+    def create(self, validated_data):
+        """Optimized bulk create operation."""
+        # Use bulk_create for better performance
+        ModelClass = self.child.Meta.model
+        instances = [ModelClass(**attrs) for attrs in validated_data]
+
+        # Bulk create with ignore_conflicts for better performance
+        try:
+            return ModelClass.objects.bulk_create(instances, ignore_conflicts=False)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)